test-integration/test_integration/test_redis_queue.py

```diff
@@ -126,11 +183,11 @@ def capture_final_response(request):
         assert waiting.result
 
         assert re.match(
-            r"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{6}Z",
+            r"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{6}\+00:00",
```

Comment:
Only a nit really but maybe we should make this `(Z|\+00:00)` as I'm 90% sure both are accepted by downstream systems.